pkg/kubelet/prober/prober.go

```diff
@@ -70,18 +70,8 @@ func newProber(
 	}
 }
 
-// recordContainerEvent should be used by the prober for all container related events.
-func (pb *prober) recordContainerEvent(pod *v1.Pod, container *v1.Container, eventType, reason, message string, args ...interface{}) {
-	ref, err := kubecontainer.GenerateContainerRef(pod, container)
-	if err != nil {
-		klog.ErrorS(err, "Can't make a ref to pod and container", "pod", klog.KObj(pod), "containerName", container.Name)
-		return
-	}
-	pb.recorder.Eventf(ref, eventType, reason, message, args...)
-}
-
 // probe probes the container.
```

Comment:
please include output explanation as we now have three values

Reply:
done